### DISCUSSION TITLE
Aerospace not recognizing when I close a window

### DISCUSSION THREAD

**Discussion by noahdurbin** [nikitabobko/AeroSpace#723] [questions-and-answers]
When I have two windows in one space and close one of these windows the other window does not tile to fit fullscreen, only when I quit the other app does it tile to the rest of the space like it should. I am on MacOS 15.1. I've tried uninstalling aerospace, going to the default config, not sure what to try next, wondering if anyone else has experienced this and if you know how I could fix it, Thanks!

**Comment by gldtn** [comment]
Maybe try running debug-windows?: https://nikitabobko.github.io/AeroSpace/commands#debug-windows
